### ISSUE TITLE
[hlx up] does not render remote sample content

### ISSUE THREAD

**Issue by kptdobe** [adobe/helix-cli#303]
**Description**
Impossible to get remote content served in my project.

**To Reproduce**

1. `hlx demo --type full test`
2. `cd test`
3. Edit `helix-config.yml` and uncomment `contentRepo` line
4. `hlx up`

Warning and error message 

```
warn: helix-config.yaml: 'contentRepo' is deprecated. Use 'content' instead.
Invalid URL: no valid git-url: https://github.com/Adobe-Marketing-Cloud/reactor-user-docs
```

5. Rename `contentRepo` to `content`
6. Rename `https://github.com/Adobe-Marketing-Cloud/reactor-user-docs` to `https://github.com/Adobe-Marketing-Cloud/reactor-user-docs.git`
7. `hlx up`
8. Open http://localhost:3000/README.html

-> white page

Warning message says:

```
[hlx] warn: tapping failed: Error: Invalid Context at step 2
data.content.mdast.children[5].start should be integer undefined, data.content.mdast.children[9].start should be integer undefined, data.content.mdast.children[15].start should be integer undefined, data.content.mdast.children[21].start should be integer undefined, data.content.mdast.children[24].start should be integer undefined, data.content.mdast.children[27].start should be integer undefined, data.content.mdast.children[32].start should be integer undefined, data.content.mdast.children[36].start should be integer undefined, data.content.mdast.children[70].start should be integer undefined, data.content.mdast.children[72].start should be integer undefined
```

**Expected behavior**

1. `contentRepo` should be `content` ootb
2. sample link should be correct
3. no warning message should be shown
4. some content needs to be rendered

**Screenshots**

**Version:**
run: `$ hlx --version`
0.8.1-pre.26 but with latest hypermedia-pipeline (@adobe/hypermedia-pipeline@0.7.3-pre.0)

**Comment by kptdobe** [comment]
https://github.com/adobe/helix-cli/issues/289 should prevent from this kind of issues.

**Comment by tripodsan** [comment]
I think this is now fixed in @0.8.1 ?
